src/components/uploader/css/Uploader.css

```diff
@@ -485,6 +485,10 @@ html>body .fl-uploader span.fl-uploader-browse {
     display:block;
 }
 
+.fl-uploader-no-js-note {
```

Comment:
I think this class should probably be renamed. 

* Look at the [naming conventions](https://wiki.fluidproject.org/display/docs/FSS+Naming+Conventions). I think what you have here is quite close to the naming structure.
* The name seems specific to the case where JS isn't enabled, but also is used when HTML5 features aren't present. This should have a more generic name like `fl-uploader-basic-note` or `fl-uploader-basic-message`